packages/create-vite/template-lit-ts/index.html

```diff
@@ -2,14 +2,15 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8" />
-    <link rel="icon" type="image/svg+xml" href="/src/favicon.svg" />
+    <link rel="icon" type="image/svg+xml" href="/vite.svg" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Vite + Lit App</title>
+    <title>Vite + Lit + TS</title>
+    <link rel="stylesheet" href="./src/index.css" />
     <script type="module" src="/src/my-element.ts"></script>
   </head>
   <body>
     <my-element>
-      <p>This is child content</p>
+      <h1>Vite + Lit</h1>
```

Comment:
```suggestion
      <h1>Vite + Lit + TS</h1>
```

Reply:
I chose to avoid the + TS in the h1 as it looked unbalanced in most templates. Better to align with others I think. It is in the titles though

Reply:
Oh this is `<h1>`. I overlooked it as `<title>`. My bad. 🤦‍♂️

Then it's perfect!